RecoHI/HiTracking/python/hiMixedTripletStep_cff.py

```diff
@@ -1,206 +1,159 @@
 import FWCore.ParameterSet.Config as cms
-
-# NEW CLUSTERS (remove previously used clusters)
-hiMixedTripletClusters = cms.EDProducer("HITrackClusterRemover",
-                                        clusterLessSolution= cms.bool(True),
-                                        oldClusterRemovalInfo = cms.InputTag("hiLowPtTripletStepClusters"),
-                                        trajectories = cms.InputTag("hiLowPtTripletStepTracks"),
-                                        overrideTrkQuals = cms.InputTag('hiLowPtTripletStepSelector','hiLowPtTripletStep'),
-                                        TrackQuality = cms.string('highPurity'),
-                                        pixelClusters = cms.InputTag("siPixelClusters"),
-                                        stripClusters = cms.InputTag("siStripClusters"),
-                                        Common = cms.PSet(
-    maxChi2 = cms.double(9.0),
-    ),
-                                        Strip = cms.PSet(
-    maxChi2 = cms.double(9.0),
-    #Yen-Jie's mod to preserve merged clusters
-    maxSize = cms.uint32(2)
-    )
-                                        )
-
-
-                                        
-
+from RecoTracker.IterativeTracking.MixedTripletStep_cff import *
+from HIPixelTripletSeeds_cff import *
+from HIPixel3PrimTracks_cfi import *
+
+#cluster remover
+hiMixedTripletStepClusters = cms.EDProducer("HITrackClusterRemover",
+     clusterLessSolution = cms.bool(True),
```

Comment:
oldClusterRemovalInfo was removed.
It's present in the non-HI version.
Was this removal intentional?

Reply:
Yes